.travis.yml

```diff
@@ -15,6 +15,7 @@ before_install:
   - docker-compose -f ./docker/docker-compose.yml up -d
 
 script:
+  - docker-compose -f ./docker/docker-compose.yml run meteoroid /bin/sh -c "pip install flake8 && flake8 ./meteoroid"
```

Comment:
Is `pip install flake8` described dev-packages section in the Pipfile, but is it required here?

Reply:
Yes, you are right. I'd rather not install the other packages of dev-packages, because   I would like to have avoided lowering speed about CI. But I definitely thought version of flake8 should be locked. 

I updated. Please check.